src/bun.js/bindings/KeyObject.cpp

```diff
@@ -2943,6 +2947,150 @@ JSC_DEFINE_HOST_FUNCTION(KeyObject__SymmetricKeySize, (JSC::JSGlobalObject * glo
     return JSC::JSValue::encode(JSC::jsUndefined());
 }
 
+static EncodedJSValue doAsymmetricCipher(JSGlobalObject* globalObject, CallFrame* callFrame, bool encrypt)
+{
+    auto count = callFrame->argumentCount();
+    auto& vm = globalObject->vm();
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    if (count != 2) {
```

Comment:
```suggestion
    if (count < 2) {
```
to be more javascripty

Reply:
ah, I see 👍